src/fr/simpliquer.md

```diff
@@ -36,26 +36,24 @@ Nous testons actuellement la version alpha de Système de design GC, ce qui corr
 
 <gcds-heading tag="h2" margin-top="225">Contribuez à nos prochaines priorités</gcds-heading>
 
-Vous pouvez contribuer à l’avenir de Système de design GC. Nous recherchons activement des contributions pour les éléments suivants.
+Vous pouvez contribuer à l’avenir de Système de design GC. 
 
-<div>
-  <ul class="mb-300">
-    <li>**Des tableaux de données** pour organiser et afficher une grande quantité de données dans des rangées et des colonnes.</li>
-    <li>**Des balises** pour étiqueter, catégoriser et organiser des éléments à l’aide de mots-clés descriptifs. </li>
-  </ul>
-</div>
+Nous recherchons activement des contributions pour les éléments suivants :
+
+- **Des tableaux de données** pour organiser et afficher une grande quantité de données dans des rangées et des colonnes.</li>
+- **Des balises** pour étiqueter, catégoriser et organiser des éléments à l’aide de mots-clés descriptifs. </li>
 
 Nous nous intéressons également aux problèmes concernant les interactions avec les utilisateurs et utilisatrice ou aux solutions qui peuvent s’appliquer aux différents services du GC.
 
-Soumettez vos contributions à nos prochaines priorités directement dans GitHub, en créant <gcds-link href="{{ links.githubGetStarted }}" external>créant un compte</gcds-link>.
+Soumettez vos contributions à nos prochaines priorités directement dans GitHub, en <gcds-link href="{{ links.githubGetStarted }}" external>créant un compte</gcds-link>.
 
 Fournissez l'un des éléments suivants pour chaque composant ou modèle de page :
 
 - **Exemples de designs** : captures d'écran, prototypes, liens vers des services en ligne.
 - **Résultats de recherche** : recherches préliminaires, exemples de cas d'utilisation ou tests d'utilisabilité.
 - **Code** : code de prototype ou de production, ou implémentations du produit.
 
-<gcds-button button-role="secondary" button-type="link" href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
+<gcds-button button-role="primary" button-type="link" href="{{ links.githubCompsPriority }}" external>Contribuer sur GitHub</gcds-button>
```

Comment:
I believe this is the code Ethan and Melanie are talking about. It's changed to "primary" for the french version, but it remains "secondary" in the english one.

Reply:
Just realized that previously we were using a secondary button in the partial in case there was a primary button on the page. In keeping with this approach, "Contribute in  GitHub" could be a secondary button type and "Contact us" could  be a secondary link type.

Reply:
They should both be secondary. I'll try to change the French.